qi-lib/flow/core/compiler/deforest/syntax.rkt

```diff
@@ -1,16 +1,13 @@
 #lang racket/base
```

Comment:
RE 0ff1e172706f53dbdacbe25af7b5f41dab6a3bf5, I'd prefer a shorter subject line and more
information in the body.